### PR TITLE
added new version 4.18.2111.5-0

### PR DESCRIPTION
**Pull request by RAJU2529**
i added new version 4.18.2111.5-0  after verifying with Windows 11 Build number 22000.376,

Screenshot 2021-12-17 185956

### CODE REVIEW

**Comment by RAJU2529** [comment]
@denisebmsft ,  #7208 please verify it and merge if correct

File: microsoft-365/security/defender-endpoint/configure-extension-file-exclusions-microsoft-defender-antivirus.md [hunk 1]
```diff
@@ -311,12 +311,12 @@ To check exclusions with the dedicated [command-line tool mpcmdrun.exe](./comman
...
 Start, CMD (Run as admin)
 cd "%programdata%\microsoft\windows defender\platform"
-cd 4.18.2110-6.0 (Where 4.18.2110-6.0 is this month's Microsoft Defender Antivirus "Platform Update".)
+cd 4.18.2111-5.0 (Where 4.18.2111-5.0 is this month's Microsoft Defender Antivirus "Platform Update".)
 MpCmdRun.exe -CheckExclusion -path <path>
 ```
 
 > [!NOTE]
-> Checking exclusions with MpCmdRun requires Microsoft Defender Antivirus CAMP version 4.18.2110-6.0 (released in November 2021) or later.
+> Checking exclusions with MpCmdRun requires Microsoft Defender Antivirus CAMP version 4.18.2111-5.0 (released in  Decemmber 2021) or later.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-extension-file-exclusions-microsoft-defender-antivirus.md]:
```suggestion
> Checking exclusions with MpCmdRun requires Microsoft Defender Antivirus CAMP version 4.18.2111-5.0 (released in  December 2021) or later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RAJU2529, kindly commit the suggested change to correct the typo. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @RAJU2529

**Comment by denisebmsft** [comment]
Thank you @JohanFreelancer9

**Pull request merged**